src/server/scripts/Northrend/Naxxramas/boss_anubrekhan.cpp

```diff
@@ -37,141 +37,139 @@ enum Spells
 {
     SPELL_IMPALE                    = 28783,
     SPELL_LOCUST_SWARM              = 28785,
-    SPELL_SUMMON_CORPSE_SCRABS_5    = 29105,
-    SPELL_SUMMON_CORPSE_SCRABS_10   = 28864,
+    SPELL_SUMMON_CORPSE_SCARABS_5   = 29105,
+    SPELL_SUMMON_CORPSE_SCARABS_10  = 28864,
     SPELL_BERSERK                   = 26662
 };
 
 enum Misc
 {
-    NPC_CORPSE_SCARAB               = 16698,
-    NPC_CRYPT_GUARD                 = 16573,
+    ACHIEV_TIMED_START_EVENT        = 9891,
+    EVENT_SPAWN_CRYPT_GUARDS_1      = 0,
+    EVENT_BERSERK                   = 1,
+    EVENT_SPAWN_CRYPT_GUARDS_EXTRA  = 2,
+};
 
-    ACHIEV_TIMED_START_EVENT        = 9891
+const Position cryptguardPositions[] = {
```

Comment:
0 and 1 can be moved to a Creature Summon Group, but it's OK here since [2] is used in the struct

https://www.azerothcore.org/wiki/creature_summon_groups